libdnf/hy-package.cpp

```diff
@@ -1065,12 +1066,38 @@ dnf_package_get_advisories(DnfPackage *pkg, int cmp_type)
         if ((cmp > 0 && (cmp_type & HY_GT)) ||
             (cmp < 0 && (cmp_type & HY_LT)) ||
             (cmp == 0 && (cmp_type & HY_EQ))) {
-            advisory = dnf_advisory_new(sack, di.solvid);
-            if (libdnf::isAdvisoryApplicable(*advisory, sack)) {
-                g_ptr_array_add(advisorylist, advisory);
-            } else {
-                dnf_advisory_free(advisory);
+
+            std::vector<libdnf::AdvisoryModule> moduleList;
+
+            dataiterator_seek(&di, DI_SEEK_PARENT);
```

Comment:
Please could you add a comment what following three lines do? Also add relation with new dataiterator below would be nice.

Reply:
I have added the comment, can you check if it is detailed enough and understandable?

Reply:
Nice